testutil/sample/observer.go

```diff
@@ -337,3 +339,15 @@ func OperationalFlags() types.OperationalFlags {
 		SignerBlockTimeOffset: ptr.Ptr(time.Second),
 	}
 }
+
+func ConfirmationParams() types.ConfirmationParams {
+	randInboundCount := Uint64InRange(1, 200)
```

Comment:
please use `Uint64InRangeFromRand`

We can take randomness as an input  parameter

Reply:
added a rand argument https://github.com/zeta-chain/node/pull/3461/commits/ea3f43b2cf2fc46c60459e7487ce0ddc60000194